tests/rbac_test.go

```diff
@@ -0,0 +1,215 @@
+package tests_test
+
+import (
+	"flag"
+	"fmt"
+	"time"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ktests "kubevirt.io/kubevirt-ansible/tests"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	"kubevirt.io/kubevirt/tests"
```

Comment:
I would suggest to import it as `ktests` to be consistent with other test's modules.